docs/content/docs/dev/table/sqlClient.md

```diff
@@ -91,6 +91,40 @@ The `SET` command allows you to tune the job execution and the sql client behavi
 After a query is defined, it can be submitted to the cluster as a long-running, detached Flink job.
 The [configuration section](#configuration) explains how to declare table sources for reading data, how to declare table sinks for writing data, and how to configure other table program properties.
 
+### Key-strokes
+
+There is a list of available key-strokes in SQL client
```

Comment:
Given that we most likely have both Windows/Linux users, as well as Mac users, can we somehow make this more unform?

Reply:
Added key-strokes for Mac.
For Windows (WSL) key strokes are the same as for Linux, mentioned it in the doc explicitly